src/Func.h

```diff
@@ -509,6 +509,17 @@ class Func {
     EXPORT void compile_to_static_library(const std::string &filename_prefix, const std::vector<Argument> &args,
                                           const Target &target = get_target_from_environment());
 
+    /** Compile to static-library file and header pair once for each target;
+     * each resulting function will be evaluated (in order) via halide_can_use_target_features()
```

Comment:
I find this comment slightly confusing. Perhaps change "evaluated" to "considered"? Evaluated sounds like you're running the pipeline.